modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyDLObjectsMVCActionCommand.java

```diff
@@ -320,6 +322,21 @@ private int _getItemsCopied(
 		return dlObjectIds.length - errorMessages.size();
 	}
 
+	private Group _getRepositoryGroup(long destinationRepositoryId)
```

Comment:
@adolfopa, just some silly question from me. Wouldn't it be better to call the parameter just repositoryId, since we can pass destination and source?

In any case, we can move it forward like this

Reply:
Yes; this was an oversight.

I'll rename and push-force the change.